Sources/DriverSupport/DriverSupportUtils.swift

```diff
@@ -17,12 +17,13 @@ import class TSCBasic.Process
 import enum TSCBasic.ProcessEnv
 import struct TSCBasic.ProcessResult
 
-public class DriverSupport {
-    private var flagsMap = ThreadSafeBox<[String: Set<String>]>()
-    public init() {}
+
```

Comment:
this is the key change, it basically means the information is cached foe the entire process which IMO is the desired effect